tests/ha/multi_pod_failure/test_multi_data_pod_failure.py

```diff
@@ -710,7 +653,6 @@ def test_cont_ios_during_server_kpods_down(self):
     @pytest.mark.ha
     @pytest.mark.lc
     @pytest.mark.tags("TEST-35792")
-    @CTFailOn(error_handler)
     def test_server_data_kpods_fail_during_ios(self):
```

Comment:
can u rather create new file for combination test and add this test and all combi tests there?

Reply:
> can u rather create new file for combination test and add this test and all combi tests there?

Will take care this change in next PR as this also needs to updated with 3 different threads for individual IOs